apps/discovery_api/config/integration.exs

```diff
@@ -54,6 +54,14 @@ config :discovery_api, :elasticsearch,
               type: "keyword",
               index: true
             },
+            modifiedDate: %{
```

Comment:
I don't much about mix config, but is there any way to combine this with the runtime.exs equivalent? Kinda sucks having this in two places

Reply:
Might be possible with the new elixir config standard.